contacts/html-helper.js

```diff
@@ -197,11 +216,20 @@ export const notificationModal = (text) => {
   document.body.appendChild(notification);
   setTimeout(() => {
     notification.remove();
-  }, 3000);
+  }, 5000);
 };
 
+export const deletedSuccessfully = (person = null) => {
+  if (person) {
```

Comment:
you can create a variable here with the text of the message and call it just once.
also, i don't like very much the way you did your logic. if person is not null then you delete multiple persons...better use a boolean value deleteMultiple